src/test/resources/capabilities.yml

```diff
@@ -39,18 +38,8 @@ cloudDriver:
       browser_version: latest
       capabilities:
         browserstack.selenium_version: 3.141.59
-#    - name: OSX_BigSur_Chrome_Latest
-#      os: OS X
-#      os_version: Catalina
-#      browser: Safari
-#      browser_version: '13.0'
```

Comment:
Shouldn't we add it back.

Reply:
we could... wasn't sure on why we commented out. hence removed for now.